arangod/Agency/MoveShard.cpp

```diff
@@ -842,6 +847,22 @@ JOB_STATUS MoveShard::pendingLeader() {
 }
 
 JOB_STATUS MoveShard::pendingFollower() {
+
+  auto considerCancellation = [&]() -> bool {
```

Comment:
Do we really need two such closures and not just one standalone function? After all, we had only one `considerTimeout` as far as I can see.